src/components/control-panel.tsx

```diff
@@ -29,68 +31,78 @@ export const ControlPanel: React.SFC<IControlPanelProps> = (props) => {
+          <Button className="start-sensor control-panel-button"
+                  onClick={props.onStartCollecting}
+                  disabled={disableStartCollecting}>
             Start
           </Button>
         );
-  
+
   function handleDurationChange(evt:React.FormEvent<HTMLSelectElement>) {
     if (props.onDurationChange)
       props.onDurationChange(Number(evt.currentTarget.value));
   }
 
-  function renderEmbedInCodapUrl(url:string|null) {
-    if (!url) return null;
-    return (
-      <a className="embed-codap-link" href={url}>Embed in CODAP</a>
-    );
-  }
-
   return (
-    <div className="control-panel">
-      <div className="cc-logo" />
-      <span className="duration-label">Duration:</span>
+    <div className={controlPanelClass}>
+
+      <div className="left-controls">
+        <div className="icon-container" onClick={props.onReloadPage}>
+          <svg className="icon reload">
+            <use xlinkHref="./assets/images/icons.svg#icon-reload" />
+          </svg>
+          <div className="icon-label">Reload</div>
+        </div>
+      </div>
+      <span className={durationLabelClass}>Duration:</span>
       <Select className="duration-select control-panel-select"
-              onChange={handleDurationChange} value={String(props.duration)}>
+              onChange={handleDurationChange}
+              value={String(props.duration)}
+              disabled={disableDuration}>
         {[durationOptions]}
       </Select>
       {props.isConnectorAwake ? startCollectingButton : startConnectingButton}
-      <Button className="stopSensor control-panel-button"
-              onClick={props.onStopCollecting} disabled={disableStopCollecting}>
+      <Button className="stop-sensor control-panel-button"
+              onClick={props.onStopCollecting}
+              disabled={disableStopCollecting}>
         Stop
       </Button>
-      <Button className="sendData control-panel-button"
-              onClick={props.onSaveData} disabled={disableSendData}>
+      <Button className="send-data control-panel-button"
+              onClick={props.onSaveData}
+              disabled={disableSendData}>
         Save Data
       </Button>
-      <Button className="newData control-panel-button"
-              onClick={props.onNewRun} disabled={disableNewData}>
+      <Button className="new-data control-panel-button"
+              onClick={props.onNewRun}
+              disabled={disableNewData}>
         New Run
       </Button>
       <div className="right-controls">
-        <div>
-          <a onClick={props.onReloadPage}
-              className="reload-page-button"
-              title="Reload All">
-              <i className="fa fa-repeat fa-2x"></i>
-          </a>
+        <div className="icon-container"
+             onClick={props.onAboutClick}>
+          <svg className="icon about">
+            <use xlinkHref="./assets/images/icons.svg#icon-about"/>
```

Comment:
@kswenson any ideas what the correct path should be? Just about anything seems to work locally, but things are hairier when running on the server. This relative path currently in use works for the main page, but it causes trouble when trying to load from the example pages (which are in a different directory and hence can't resolve the path to icons.svg - they need ../assets/images/icons.svg to move up a directory first). Using / for the root directory didn't seem to work. Should this icons.svg file live somewhere else? Is there a Webpack setting that we need?

Reply:
It looks like the assets path is used twice in `control-panel.tsx` and once in `graph-top-panel.tsx`. For Webpack to be the solution, I think we'd have to configure Webpack to include SVGs in the bundle. My inclination for a relatively quick fix would be to:
1. Pass an `assetsPath` property into the components that need it.
2. Centralize the determination of the `assetsPath` property somewhere fairly high up in the source code hierarchy, e.g. `App.tsx`.
3. At that point, determine the value of the `assetsPath` by parsing `window.location` for `/examples/` so that the default value is what works for the main page, but the alternate path can be used for the examples.
4. Pass the `assetsPath` down as a prop
5. Alternatively, one could conceivably introduce React context here, but I think the component hierarchy is shallow enough that passing `assetsPath` down explicitly wouldn't be too bad.